lib/europeana/api/record.rb

```diff
@@ -8,6 +8,8 @@ module API
     class Record
       include Resource
 
+      autoload :LangMap, 'europeana/api/record/lang_map'
```

Comment:
Why autoload here? Is this just to specify the file path?

Reply:
Yes. This just tells Ruby where the module is defined, so that when elsewhere reference is made to `Europeana::API::Record::LangMap`, it can autoload the right file.

Reply:
then why not make LangMap a concern and include it like Resource above for consistency?